pkgs/tools/misc/lazydocker/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "lazydocker-${version}";
```

```suggestion
  pname = "lazydocker";
```

And renove `rec`

---

> ```suggestion
>   pname = "lazydocker";
> ```

:+1: 

> And renove `rec`

If I remove `rec` from the argument to `buildGoModule`, I won't anymore be able to use
```nix
"v${version}"
```
for `rev` below.

Please advise.

---

Oh right, `rec` stays then.

---

switched to `pname` in 1658e84